### PR TITLE
🌐 Update Urdu translation for `docs/ur/docs/benchmarks.md`

### PR DESCRIPTION
**Pull request by AhsanSheraz**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit b0efbaaceb13c1da34283977d3c9d5959f784880 at: https://3fc208db.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 641e549d254b821d13938baeec4e7b5822aa70b4 at: https://8aa88612.fastapitiangolo.pages.dev

**Comment by talhaumer** [comment]
Looks Good

**Comment by AhsanSheraz** [comment]
@tiangolo  Can you please add labels to this MR?

**Review by sulemanhelp — APPROVED** [review]

**Review by aimasheraz1 — APPROVED** [review]

**Review by talhaumer — APPROVED** [review]
Looks good

**Comment by talhaumer** [comment]
Looks good

**Review by talhaumer — APPROVED** [review]
Looks good

**Review by talha-softversum — APPROVED** [review]
Looks Good

File: docs/ur/docs/benchmarks.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# بینچ مارکس
+# <div style="text-align: right;">بینچ مارکس</div>
```
Comment (tiangolo) [inline docs/ur/docs/benchmarks.md:1]: Is this necessary? 🤔 

I would think maybe there's then a configuration missing in MkDocs... I'll have to investigate it.

Comment (AhsanSheraz) [reply docs/ur/docs/benchmarks.md:1]: @tiangolo previously it was left aligned so I have to add this you can check and let me know. Also I simply inherit the parent (en) MkDocs in my ur folder MkDocs file.

**Review by talhaumer — APPROVED** [review]
Looking Good

**Review by tiangolo — CHANGES_REQUESTED** [review]
Hello! I discovered there's was a bug in how I set up MkDocs Material, that's why all the CSS styles to set up text right-to-left were needed. Now that it's fixed in https://github.com/tiangolo/fastapi/pull/10490 none of that should be needed.

I also updated how this page is structured here: https://github.com/tiangolo/fastapi/pull/10495

So now the content to translate should be smaller, e.g. no translations needed for each language name, as all that is taken from the config. In that same PR I also updated the existing languages, you could do the same update here. 🙏 

Thanks! ☕

**Comment by tiangolo** [comment]
📝 Docs preview for commit 143104b927476065471b3005920563fe8a66f63b at: https://839e56a4.fastapitiangolo.pages.dev

**Comment by AhsanSheraz** [comment]
> Hello! I discovered there's was a bug in how I set up MkDocs Material, that's why all the CSS styles to set up text right-to-left were needed. Now that it's fixed in #10490 none of that should be needed.
> 
> I also updated how this page is structured here: #10495
> 
> So now the content to translate should be smaller, e.g. no translations needed for each language name, as all that is taken from the config. In that same PR I also updated the existing languages, you could do the same update here. 🙏
> 
> Thanks! ☕

@tiangolo fixed the things that you told.

**Review by aimasheraz1 — APPROVED** [review]

**Review by talhaumer — APPROVED** [review]

**Review by talhaumer — APPROVED** [review]
Looking Good

**Review by tiangolo — CHANGES_REQUESTED** [review]
Thanks @AhsanSheraz! 

Now that the RTL bug is fixed, please remove all the extra `style="direction: rtl;"` and their divs. Thanks! ☕

**Comment by AhsanSheraz** [comment]
> Thanks @AhsanSheraz!
> 
> Now that the RTL bug is fixed, please remove all the extra `style="direction: rtl;"` and their divs. Thanks! ☕

Done @tiangolo

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6929a7069762762276c569b2c28650df80c236d1 at: https://86521267.fastapitiangolo.pages.dev

**Comment by AhsanSheraz** [comment]
@tiangolo everything is working good but we have to add this <div style="text-align: right;"> because it is making align to left but it should be at right, so adding this as well.
image

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2b1c77485bc98c5ae2a443f6271e17eb84e0c741 at: https://5ff66add.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
I just noticed that there was another CSS bug introduced by a previous RTL PR, I just fixed it here: https://github.com/tiangolo/fastapi/pull/11039

Now with this, you can remove all the `style="text-align: right;"` blocks. 🤓

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6cebf0599cc2daf70d99e96bb016b3f6c2b26c27 at: https://05a3595f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 82a9dd4592d029860adbb3ca7d59a6c6a5e12626 at: https://1e425d33.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6c9362e268ff48273b53f379fc56323f05ba2d29 at: https://28dd6e57.fastapitiangolo.pages.dev

**Review by AhsanSheraz — COMMENTED** [review]
Fixed all the changes in this [MR](https://github.com/tiangolo/fastapi/pull/10046)

**Comment by AhsanSheraz** [comment]
> I just noticed that there was another CSS bug introduced by a previous RTL PR, I just fixed it here: #11039
> 
> Now with this, you can remove all the `style="text-align: right;"` blocks. 🤓

@tiangolo Fixed all the changes in this MR related to `style="text-align: right;" divs and removed this from all the files.

**Review by aimasheraz1 — APPROVED** [review]
Looks good now without style and div

**Comment by tiangolo** [comment]
Thank you! :nerd_face: 

We are no longer translating some specific pages: https://fastapi.tiangolo.com/contributing/#dont-translate-these-pages

So I updated it to include the updated page that would go in. Thanks! :cake:

**Pull request merged**